chpl/chpl-service/src/main/java/gov/healthit/chpl/changerequest/dao/ChangeRequestDeveloperDemographicsDAO.java

```diff
@@ -82,24 +84,24 @@ private ChangeRequestDeveloperDetailsEntity getNewEntity(ChangeRequest cr, Chang
         return entity;
     }
 
-    private ChangeRequestDeveloperDetailsEntity getEntity(Long changeRequestDevId)
+    private ChangeRequestDeveloperDemographicsEntity getEntity(Long changeRequestDevId)
             throws EntityRetrievalException {
-        String hql = "FROM ChangeRequestDeveloperDetailsEntity crDev "
+        String hql = "FROM ChangeRequestDeveloperDemographicsEntity crDev "
                 + "JOIN FETCH crDev.changeRequest "
                 + "WHERE (NOT crDev.deleted = true) "
                 + "AND (crDev.id = :changeRequestDevId) ";
 
-        List<ChangeRequestDeveloperDetailsEntity> result = entityManager
-                .createQuery(hql, ChangeRequestDeveloperDetailsEntity.class)
+        List<ChangeRequestDeveloperDemographicsEntity> result = entityManager
+                .createQuery(hql, ChangeRequestDeveloperDemographicsEntity.class)
                 .setParameter("changeRequestDevId", changeRequestDevId)
                 .getResultList();
 
         if (result == null || result.size() == 0) {
             throw new EntityRetrievalException(
-                    "Data error. Change request developer details not found in database.");
+                    "Data error. Change request developer demographics not found in database.");
         } else if (result.size() > 1) {
             throw new EntityRetrievalException(
-                    "Data error. Duplicate change request developer details in database.");
+                    "Data error. Duplicate change request developer demographic in database.");
```

Comment:
The error on line 101 has `demographics`. This one is singular
```suggestion
                    "Data error. Duplicate change request developer demographics in database.");
```